store/src/server/mod.rs

```diff
@@ -12,7 +12,7 @@ mod api;
 // STORE INITIALIZER
 // ================================================================================================
 
-#[instrument(target = "miden-store", skip_all)]
+#[instrument(target = "miden-store", name = "store::serve", skip_all)]
```

Comment:
Should we change the name just to `store` here? If I'm understanding how this works correctly, this will mean that we get `store` prefix on all requests to the store and then maybe we don't need to prefix them with `store` in other places?